cmd/podman/varlink/io.podman.varlink

```diff
@@ -1098,6 +1115,9 @@ method ContainerRestore(name: string, keep: bool, tcpEstablished: bool) -> (id:
 # ContainerRunlabel runs executes a command as described by a given container image label.
 method ContainerRunlabel(runlabel: Runlabel) -> ()
 
+# ContainerExecute runs executes a command in the given container.
+method ContainerExecute(opts: ExecOpts) -> ()
```

Comment:
Rename to `ContainerExec`

Reply:
Also - shouldn't this return something? Error code or similar?

Reply:
I got this code from @jwhonce 's PR. I actually don't think it's done. I just used it because I needed it for cross compilation.